src/tests/ftest/util/test_utils_container.py

```diff
@@ -403,19 +399,16 @@ def create_snap(self, snap_name=None, epoch=None):
                 "Error: Undefined control_method: %s",
                 self.control_method.value)
 
-        self.epoch = output.split()[1]
+        self.epoch = data["epoch"]
 
     @fail_on(DaosApiError)
-    def destroy_snap(self, snap_name=None, epoch=None):
+    def destroy_snap(self, snap_name=None, epc=None, epcrange=None):
         """Destroy Snapshot using daos utility
 
         Args:
-            pool (str): pool uuid
-            cont (str):Cont uuid
             snap_name (str, optional): Snapshot name
-            epoch (str, optional): Epoch ID
-            svc (str, optional): Pool svc
-            sys_name (str, optional): system name
+            epc (str): Epoch ID that indicates the snapshot to be destroyed.
+            epcrange (list, optional): list of Epoch IDs as strings
```

Comment:
Are handling this as a string or list?  The daos command requires a string, so we should either pass in a string, `"-".join(epoch_list[::len(epoch_list) - 1])` - my vote -, or do the conversion inside one of the methods calling the daos command.

Reply:
I updated both of the methods, container_destroy_snap and destroy_snap, to take epcrange as string and updated the test to build the formatted range in string.